docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md

```diff
@@ -644,27 +646,39 @@ module.exports = {
 
 ## Migrating `admin`
 
-Customizing the admin is as simple as creating a file in the `./admin` folder of your app. You need to make sure the file you want to customize is at the same location in your `./admin` folder as it is in the `strapi-admin` package. For a reference you can look at the [source code](https://github.com/strapi/strapi/tree/master/packages/strapi-admin/admin).
+For the beta there are quite a lot of changes made to the admin:
+
+- If you have not customized anything in `./admin` folder, then you simply delete the `./admin` folder and it's contents. 
+- If you have customized any part of the `./admin` folder, then you need to keep only those modified files, locate their new location in the directory structure ([source code](https://github.com/strapi/strapi/tree/master/packages/strapi-admin/admin)), and then move it to the new location.   
```

Comment:
conjugaison issues teir locition / move it ?

Reply:
Is this french or did you just bang you head on the keyboard? 😆

Reply:
This is corrected. @alexandrebodin .
@derrickmehaffy - franglais is a language...